### PR TITLE
polish the new SK website

### PR DESCRIPTION
**Pull request by queser9**
# Description
 
1，added social media hyperlinks and icon in the footer
2 ，make the icon responsive to theme changes
3 ，added the hyperlinks to out core team and contributors
 
## Changes Made

 
# How Has This Been Tested?
 
## Testing Checklist:

- [x] Prettier
- [x] Vale
 
# Checklist
 
Please make sure you have completed the following before merging this pull request:
 
- [x] My submission follows the style guidelines of this project
- [x] My submission is based on the relevant template in `Instructions and Templates/`
- [x] My tutorial is based on an approved tutorial proposal
- [x] I have requested a review from Seniors on the Pull Request

### CODE REVIEW

**Comment by queser9** [comment]
[Update Landing Page Footer and Add Facebook Icon](https://trello.com/c/fky6r3uc/320-update-landing-page-footer-and-add-facebook-icon)

**Comment by omckeon** [comment]
Resolve conflicts, and then you can move the Trello card into Mentor review

File: astro.config.mjs
Comment (omckeon) [inline astro.config.mjs:1]: SplashKit does not have a facebook page, so this is not needed. I would also suggest removing twitter, as the account is not being used anymore it seems.

File: src/components/social.astro
Comment (omckeon) [inline src/components/social.astro:1]: SplashKit does not have a facebook page, so this is not needed. I would also suggest removing twitter, as the account is not being used anymore it seems.

File: src/content/docs/index.mdx
Comment (omckeon) [inline src/content/docs/index.mdx:1]: SplashKit does not have a facebook page, so this is not needed. I would also suggest removing twitter, as the account is not being used anymore it seems.

The links in the footer are good though.

File: src/styles/custom.css
Comment (omckeon) [inline src/styles/custom.css:1]: As the icons are included in the top right corner of the page, these icons are not needed.

**Review by omckeon — CHANGES_REQUESTED** [review]
See the comments in each file to see the changes requested.
Also, make sure to do 'git pull' before updating, as there seemed to be some old code when I tested these changes.

**Comment by omckeon** [comment]
Closing this PR to simplify changes.

**Pull request closed without merging**